src/util/shell.js

```diff
@@ -1,19 +1,29 @@
 import execa from 'execa';
+import { execSync } from 'child_process';
 
-export async function exec(commands) {
+export async function exec(commands, std = `stdout`) {
   try {
     if (typeof commands === 'string') {
       commands = [commands];
     }
     let output = '';
     for (let command of commands) {
       const [first, ...rest] = command.match(/".+"|\S+/g);
-      output = (await execa(first, rest)).stdout;
+      const execResult = await execa(first, rest);
+      output = std == 'stderr' ? execResult.stderr : execResult.stdout;
     }
     return output;
-  } catch(err) {
+  } catch (err) {
     const { stderr, stdout } = err;
     const message = (stderr || stdout).split('\n').join(' ');
     throw new Error(message);
   }
 }
+
+export async function execSyncInherit(command) {
```

Comment:
I'm not familiar with `stdio: inherit` ... is there any way we could fold this into `exec`? If not maybe document when/how to use this function?

Reply:
For example, when you are pushing your docker image to gcr.io, you see progression animations, that you wouldn't see if you use the `exec` that only captures the end result (final output string). stdio `inherit` does a pass through of the output to the current (node.js) process, so you can see the live progression/status output.

I'll add documentation.

Reply:
ahh ok this totally makes sense then... I guess `cloud` will be fundamentally different from other commands where the user is going to actually care about the details of what's going on whereas other commands we can/should hide the bedrock-specific details...